app/pages/multisig/updates/BakerStakeThreshold/UpdateBakerStakeThreshold.tsx

```diff
@@ -23,13 +22,12 @@ export default function UpdateBakerStakeThreshold({
         blockSummary.updates.chainParameters.minimumThresholdForBaking;
 
     return (
-        <>
-            <div className="body1">
-                <Label>Current baker stake threshold (µCCD)</Label>
-                {currentBakerStakeThreshold.toString()}
+        <div>
+            <div className="body3 mono mB10">
+                Current threshold: {currentBakerStakeThreshold.toString()}
```

Comment:
I think we should keep the unit.

Reply:
Before:
![Screenshot from 2022-01-04 15-26-19](https://user-images.githubusercontent.com/87635671/148073671-f0c150f2-e0bd-4337-bf12-0f3bd04655c1.png)
Current:
![Screenshot from 2022-01-04 15-26-27](https://user-images.githubusercontent.com/87635671/148073668-550aa1c9-f529-4a47-984a-adecfdb80f24.png)